scio-examples/src/main/scala/com/spotify/scio/examples/extra/CloudSQLExample.scala

```diff
@@ -0,0 +1,90 @@
+/*
+ * Copyright 2017 Spotify AB.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied.  See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ */
+
+package com.spotify.scio.examples.extra
+
+import java.sql.{PreparedStatement, ResultSet}
+
+import com.spotify.scio.jdbc._
+import com.spotify.scio.{ContextAndArgs, ScioContext}
+import org.apache.beam.examples.common.ExampleCloudSQLOptions
+import org.apache.beam.sdk.values.KV
+
+// Read from google Cloud SQL database table and write to different table in the same database.
+object CloudSQLExample {
+  def main(cmdlineArgs: Array[String]): Unit = {
+    // Ensure this has been called with required database connection details.
+    val (cloudSqlOptions, extraArgs) =
+      ScioContext.parseArguments[ExampleCloudSQLOptions](cmdlineArgs)
+
+    val (sc, args) = ContextAndArgs(cmdlineArgs)
+    val connOpt = getDbConnectionOptions(cloudSqlOptions)
+    val readOptions = getReadOptions(connOpt)
+    val writeOptions = getWriteOptions(connOpt)
+
+    sc.withName("Read from Cloud Sql").jdbcSelect(readOptions)
```

Comment:
Do you really need the `withName`s here? It's not a streaming job and simpler without.